src/preview/rewriteStyleSheet.ts

```diff
@@ -67,11 +119,11 @@ const rewriteRule = ({ cssText, selectorText }: CSSStyleRule, shadowRoot?: Shado
 // A sheet can only be rewritten once, and may carry over between stories.
 export const rewriteStyleSheet = (
   sheet: CSSStyleSheet,
-  shadowRoot?: ShadowRoot
+  forShadowDOM = false
```

Comment:
We never actually needed the `ShadowRoot` -- a boolean suffices and makes test writing easier.